src/nft/components/details/detailsV2/LandingPage.tsx

```diff
@@ -40,43 +44,40 @@ const InfoContainer = styled(ColumnCenter)`
   }
 `
 
+const StyledHeadlineText = styled(ThemedText.HeadlineSmall)``
+const StyledSubheaderText = styled(ThemedText.SubHeaderSmall)`
+  line-height: 20px;
+`
+
 const InfoDetailsContainer = styled(Column)`
   gap: 4px;
   align-items: center;
```

Comment:
textAlign should also be centered, looking at the subheader left aligned below
![Screen Shot 2023-05-08 at 11 38 33 ](https://user-images.githubusercontent.com/11512321/236904671-7d365020-3fcc-462a-acc5-c1eb1c304f04.png)